alpha/experience-platform-extension/experience-platform-setup.md

```diff
@@ -0,0 +1,24 @@
+# Experience Platform setup
+
+## Prepare Platform
+
+To send data to [Adobe Experience Platform](https://platform.adobe.com/), you need to create an XDM schema and a dataset that uses the schema.
+
+- [Create a schema](https://www.adobe.io/apis/experienceplatform/home/tutorials/alltutorials.html#!api-specification/markdown/narrative/tutorials/schema_editor_tutorial/schema_editor_tutorial.md)
+- Assign the XDM ExperienceEvent class to your schema.
+- Add the following mixins: 
+  - ExperienceEvent Application Details
+  - ExperienceEvent Environment Details
+  - ExperienceEvent Commerce Details
+- [Create a dataset](https://platform.adobe.com/dataset/overview) where your data should be sent by using the schema that you created earlier.
+
+## Requesting a configuration ID
+
+To use the Mobile SDK, you need a configuration ID, which ensures that your data is routed to the correct location. To obtain a configuration ID from your consultant or through Adobe Client Care, provide the following information:
+
+- Org ID: To locate this ID, complete the instructions in [Organizations and account linking](https://docs.adobe.com/content/help/en/core-services/interface/manage-users-and-products/organizations.html).
```

Comment:
The formatting of this list appears a little cluttered. Maybe use dashes '-' to separate the config key with its description to provide more space between them?

- Org ID - To locate this ID, complete the instructions in [Organizations and account linking](https://docs.adobe.com/content/help/en/core-services/interface/manage-users-and-products/organizations.html).
- Dataset ID - Available in the dataset UI when you click on a dataset.
- Schema ID - Available at the end of the URL of the schema creation/viewing page.
- Friendly Name - The name that will be used in future UIs for this configuration.

Reply:
Updated based on @rekrishn 's suggestions, will ask for her advice before changing them back.